src/sticky.js

```diff
@@ -107,11 +110,19 @@ export default class Sticky {
     return absoluteBottom - paddingBottomPixels;
   }
 
-  static normalizeElement(value: string|HTMLElement, fallback: HTMLElement): HTMLElement {
+  static normalizeElement(value: SelectorOrElement, ...fallbacks: MaybeHTMLElement[]): HTMLElement {
     if (value instanceof HTMLElement) {
       return value;
     }
-    return document.querySelector(value) || fallback;
+
+    const element: ?HTMLElement = ([document.querySelector(value), ...fallbacks]
+      .find(item => !!item && item instanceof HTMLElement): any);
```

Comment:
👀 `Array.prototype.[filter|find]` type refinement problem workaround
https://github.com/facebook/flow/issues/1414#issuecomment-231680334

😭

Reply:
get the power of Typescript 🙄